src/battle/battle.cpp

```diff
@@ -2,7 +2,6 @@
 #include "../utils/utils.h"
 #include "foe_attack.h"
 #include "main_character_attack.h"
-
```

Comment:
No need to remove the empty line. I would argue that separating our own imported files from the std lib ones is kinda helpful.